WalletWasabi/Crypto/ZeroKnowledge/ProofSystem.cs

```diff
@@ -149,7 +152,7 @@ public static (Knowledge knowledge, IEnumerable<GroupElement> bitCommitments) Ra
 			//   randomness because the server can only give idempotent responses with
 			//   its own records.
```

Comment:
unrelated to this PR, but this comment could use clarification

```suggestion
			//   randomness because otherwise recovery of credentials from
			//   seed would not result in idempotent responses if the client
			//   loses state
```